CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Added
 
+- The OTLP exporter now has two new convenience functions, `NewExportPipeline` and `InstallNewPipeline`, setup and install the exporter in tracing and metrics pipelines. (#1348)
```

Comment:
This should be un the `Unreleased` section above, not under `0.14.0`.

Reply:
updated and merged with current master